concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+ID	NUMBER	SPANISH	ENGLISH	SWADESH_ROWE	CONCEPTICON_ID	CONCEPTICON_GLOSS
```

Comment:
@CarolinHu, the Swadesh_row indicates if the concepts appear in Swadesh, but which list? If they appear in Swadesh-1952-200 you could name the row SWADESH_1952_200 and if they are the 100 list, you can name it SWADESH_1955_100, we have some other lists in which we proceeded in this way.

Reply:
Unfortunately, the author did not specify which list(s) she took the items from in the document. Neither did she reference any Swadesh lists in the grammar that was later published.